target/linux/bmips/base-files/etc/uci-defaults/09_fix_crc

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
```

Comment:
uci-defaults files are sourced, so the should not contain a shebang.

However, it is convenient to add "exit 0" at the end of the file, because otherwise there deletion will depend on the status of the latest command.